### ISSUE TITLE
Microsoft Art Gallery and 256 colors mode

### ISSUE THREAD

**Issue by anonimuos** [otya128/winevdm#729]
Microsoft Art Gallery (https://winworldpc.com/product/microsoft-art-gallery/1) runs fine with OTVDM but animations require 256-color mode. When the compatibility flag is set on its EXE for 256 colors, the display rendering is corrupted/garbled. It runs ok only in High Color/True Color modes but animations don't play.

**Comment by cracyc** [comment]
This is very similar to https://github.com/otya128/winevdm/issues/638.  Here is uses setbitmapbits with no palette in sight.  The only workaround is to use dibs but the colors are still messed up same as if otvdm itself is set to 256 color compat mode.

**Comment by anonimuos** [comment]
Yes it's a minor issue. The animations are just a few red directional lines. Can enjoy the program without it too.
